docs/api/modules/model.rst

```diff
@@ -48,6 +48,8 @@ Bidirectional Encoder Representations from Transformers
     BERTEncoder
     BERTEncoderCell
     BERTPositionwiseFFN
+    bert_12_768_12
+    bert_24_1024_16
```

Comment:
Why we didn't add [BERT-Large, Cased](https://storage.googleapis.com/bert_models/2018_10_18/cased_L-24_H-1024_A-16.zip) this time?

Reply:
There are a few more models in the pipeline and I'll add them in the next PR